docs/notebooks/visualizing_with_tensorboard.pct.py

```diff
@@ -78,6 +78,11 @@
 trieste.logging.set_summary_filter(lambda name: True)  # enabe all summaries
 
 
+# %% [markdown]
+# Furthermore, some summaries such as pairplots also require special plotting tools, which can be installed by running `pip install trieste[plotting]`. A warning message will be printed if you try to generate a summary without the necessary tools.
```

Comment:
is this the only way to install it?

Reply:
No. You can also just install seaborn with `pip install seaborn` but if we ever require other plotting libraries it'll be easy to extend what trieste[plotting] installs.